src/app/content/studyGuides/selectors.ts

```diff
@@ -92,6 +92,17 @@ const rawSummaryLocationFilters = createSelector(
   (filters) => filters.locationIds
 );
 
+export const highlightColorFiltersWithContent = createSelector(
+  totalCountsPerPageOrEmpty,
+  (totalCounts) => getHighlightColorFiltersWithContent(totalCounts)
+);
+
+export const summaryColorFilters = createSelector(
+  summaryFilters,
+  highlightColorFiltersWithContent,
+  (filters, withContent) => new Set(filters.colors.filter((color) => withContent.has(color)))
```

Comment:
You could move this to selectorsUtils if you wanted to since those are the same for highlights and sg

Reply:
I think it is so simple that it is not worth of separate file